lineman/app/components/thread_page/previous_proposals_card/previous_proposals_card.coffee

```diff
@@ -5,8 +5,8 @@ angular.module('loomioApp').directive 'previousProposalsCard', ->
   replace: true
   controller: ($scope, $rootScope, $location, Records, ProposalFormService) ->
 
-    Records.votes.fetchMyVotes($scope.discussion.closedProposals())
     Records.proposals.fetchByDiscussion($scope.discussion).then ->
+      Records.votes.fetchMyVotes($scope.discussion.closedProposals())
```

Comment:
Really this wants to be

```
Records.votes.fetchMyVotes($scope.discussion)
```

but this fix works for now. I may take some time tomorrow to do a better fix.